packages/mobile/src/in-house-liquidity/index.ts

```diff
@@ -84,6 +84,8 @@ export interface BankAccount {
   accountName: string
   accountType: string
   accountNumberTruncated: string
+  institutionName: string
+  institutionLogo: string
```

Comment:
if `institutionLogo` is expected to be nullable, that should be reflected in the type so that we get compile-time errors wherever we forgot to use null checks